api/pkg/cli/cmd/install/install_test.go

```diff
@@ -96,7 +96,7 @@ func TestInstall_NewResource(t *testing.T) {
 
 	err := opts.run()
 	assert.NoError(t, err)
-	assert.Equal(t, "Task foo(0.3) installed in hub namespace\n", buf.String())
+	assert.Equal(t, "WARN: Pipelines version unknown\nTask foo(0.3) installed in hub namespace\n", buf.String())
```

Comment:
Should we warn here? the goal of checking the pipelines version is to check if a resource can be installed or not. here the resource is installed so I guess we don't need to warn. wdyt?

Reply:
We would also be giving out the warning in case the Pipelines version is unknown. So in this test I haven't set up the Tekton version so it will just print the warning and also install the task